src/globus_cli/parsing/shared_options/endpointish.py

```diff
@@ -89,64 +94,83 @@ def _apply_endpointish_create_or_update_params(
     verify_style: str,
+    skip: tuple[str, ...] = (),
 ) -> C:
-    decorators: list[t.Callable[[C], C]] = []
+    decorators: dict[str, t.Callable[[C], C]] = {}
 
     if display_name_style == "argument":
-        decorators.append(click.argument("DISPLAY_NAME"))
+        decorators["display_name"] = click.argument("DISPLAY_NAME")
     elif display_name_style == "option":
-        decorators.append(click.option("--display-name", help=f"Name for the {name}"))
+        decorators["display_name"] = click.option(
+            "--display-name", help=f"Name for the {name}"
+        )
     else:
         raise NotImplementedError()
 
-    decorators.extend(
-        [
-            click.option(
+    decorators.update(
+        dict(
+            description=click.option(
                 "--description", help=f"Description for the {name}", type=StringOrNull()
             ),
-            click.option(
+            info_link=click.option(
                 "--info-link",
                 help=f"Link for info about the {name}",
                 type=StringOrNull(),
             ),
-            click.option(
+            contact_info=click.option(
                 "--contact-info",
                 help=f"Contact info for the {name}",
                 type=StringOrNull(),
             ),
-            click.option(
+            contact_email=click.option(
                 "--contact-email",
                 help=f"Contact email for the {name}",
                 type=StringOrNull(),
             ),
-            click.option(
+            organization=click.option(
                 "--organization",
                 help=f"Organization for the {name}",
                 type=StringOrNull(),
             ),
-            click.option(
+            department=click.option(
                 "--department",
                 help=f"Department which operates the {name}",
                 type=StringOrNull(),
             ),
-            click.option(
+            keywords=click.option(
                 "--keywords",
                 type=str if keyword_style == "string" else CommaDelimitedList(),
                 help="Comma separated list of keywords to help searches "
                 f"for the {name}",
             ),
-            click.option(
+            default_directory=click.option(
                 "--default-directory",
                 type=StringOrNull(),
                 help="Default directory when browsing or executing tasks "
                 f"on the {name}",
             ),
-            click.option(
+            force_encryption=click.option(
                 "--force-encryption/--no-force-encryption",
                 default=None,
                 help=f"Force the {name} to encrypt transfers",
             ),
-        ]
+            user_message=click.option(
+                "--user-message",
+                help=(
+                    "A message for clients to display to users when interacting "
+                    "with this {name}"
```

Comment:
```suggestion
                    f"with this {name}"
```

Reply:
Oy vey... I keep on making this mistake lately. I think I need to slow down or drink more coffee. ... Yeah, definitely more coffee. ☕☕☕

Thank you again for spotting these.